src/simmi/exam1/ArrayInteger.java

```diff
@@ -0,0 +1,34 @@
+/*Coding Exam - 1 : 18th April 2021
+
+Program 1 : 
+Given an array of integers nums and an integer target, return indices of the two numbers such that they add up to the target.
+
+Input: nums = [2,11,15,7,4], target = 9
+
+Output: [0,3]
+
+Output: Because nums[0] + nums[3] == 9, we return [0, 3].*/
+
+package simmi.exam1;
+
+public class ArrayInteger {
+	int[] integerArray(int[] arr, int target) {
+		//int[] output = new int[arr.length];
+		for(int index = 0; index<arr.length; index++) {
+			for(int innerIndex = 1;innerIndex < arr.length; innerIndex++ ) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files